pkg/lua/Kconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config PACKAGE_LUA
+    bool "LUA language package"
+    depends on TEST_KCONFIG
+    depends on HAS_ARCH_32BIT
+
```

Comment:
In the makefile there seems to be an extra condition in `Makefile.dep` blacklisting the `picolibc` feature, but I think they meant something like:
```
depends on !MODULE_PICOLIBC
```
Not 100% sure though